reo/nested_inputs.py

```diff
@@ -109,13 +109,32 @@ def list_of_str(input):
       "type": "str",
       "description": "The unique ID of a scenario created by the REopt Lite Webtool. Note that this ID can be shared by several REopt Lite API Scenarios (for example when users select a 'Resilience' analysis more than one REopt API Scenario is created)."
     },
-	"optimality_tolerance": {
+	"optimality_tolerance_bau": {
       "type": "float",
-      "min": 0.0,
+      "min": 1.0e-5,
       "max": 10.0,
-      "default": 1.0e-3,
+      "default": 0.001,
       "description": "The threshold for the difference between the solution's objective value and the best possible value at which the solver terminates"
     },
+	"optimality_tolerance_techs": {
+      "type": "float",
+      "min": 1.0e-5,
+      "max": 10.0,
+      "default": 0.001,
+      "description": "The threshold for the difference between the solution's objective value and the best possible value at which the solver terminates"
+    },
+	"use_decomposition_model": {
```

Comment:
If we're going to allow the user to toggle the decomposition model on/off and specify the subproblem gap, I think we should also allow the subproblem time limit to be customized too.  Would you agree?